package.json

```diff
@@ -11,6 +11,7 @@
     "lint": "lerna run --stream --concurrency 1 lint"
   },
   "workspaces": [
+    "packages/identity-manager",
```

Comment:
Maybe we should name it just to be `identify`?

Reply:
maybe just `identity`, but I feel like the confusion would be between it and identify

Reply:
maybe, I will leave the decision to you.